### PR TITLE
Market: replaced 'install' with 'download'

### PR DESCRIPTION
**Pull request by LudoHolbik**
## Description of changes

- replaced _install_ with _download_ (and respective word variations)
- implemented in multiple places across addons market (not just button)
- replaced "Not installed" section with "Downloads available" ( since we already have "Updates Available")
- replaced icons `upgrade` or `download_for_offline` with `download`

### Additional context

https://github.com/ynput/ayon-frontend/assets/16327908/d8e8f2a0-237f-4458-9af5-41d52819b934

### CODE REVIEW

File: src/pages/MarketPage/AddonDetails/AddonDetails.jsx [hunk 1]
```diff
@@ -106,14 +106,14 @@ const AddonDetails = ({ addon = {}, isLoading, onInstall, isUpdatingAll }) => {
     actionButton = (
       <Button
         variant="filled"
-        icon={'upgrade'}
+        icon={'download_for_offline'}
```
Comment (LudoHolbik) [inline src/pages/MarketPage/AddonDetails/AddonDetails.jsx:109]: relates to what @Innders wrote here, hope I understood it correctly https://discord.com/channels/517362899170230292/1240288129001328640/1240959874460811375

Comment (LudoHolbik) [reply src/pages/MarketPage/AddonDetails/AddonDetails.jsx:109]: image

Comment (LudoHolbik) [reply src/pages/MarketPage/AddonDetails/AddonDetails.jsx:109]: No i didnt, there should always be download.

**Comment by LudoHolbik** [comment]
@Innders You can review, I have made one change after request becase I misunderstood one comment from you. 

This is final PR version.

File: src/components/MarketAddonCard/MarketAddonCard.jsx
Comment (Innders) [inline src/components/MarketAddonCard/MarketAddonCard.jsx:1]: Clicking on Download does not work.

image

**Review by Innders — CHANGES_REQUESTED** [review]
One issue, otherwise all good.

File: src/components/MarketAddonCard/MarketAddonCard.jsx
Comment (LudoHolbik) [reply src/components/MarketAddonCard/MarketAddonCard.jsx:1]: Fixed, pushed.

**Review by Innders — APPROVED** [review]

**Pull request merged**